docs_robots/outdoor_robots/husky/a300/integration_husky/integration_husky.mdx

```diff
@@ -414,18 +414,20 @@ inputs can be driven by one of the following:
 - a 24 V input
 - a push button or switch
 
-In turn, the state of the input can be read from the ROS topic `/a300_#####/platform/mcu/status/aux_inputs`.
-The AUX inputs are active low and are mapped to the ROS message bits as follows:
+In turn, the state of the inputs can be read from the ROS topic `/a300_#####/platform/mcu/status/pinout`.
```

Comment:
Inputs, outputs, and power rails

Reply:
Since this subsection was just for the AUX inputs, I was just focusing on those here. Similar callouts to this topic are in other sections of the page for those functions, where needed.